functionaltests/single_upgrade_test.go

```diff
@@ -50,6 +47,9 @@ type additionalFunc func(t *testing.T, ctx context.Context, esc *esclient.Client
 type singleUpgradeTestCase struct {
 	fromVersion ecclient.StackVersion
 	toVersion   ecclient.StackVersion
+	// deployStandalone determines whether to deploy the cluster in
+	// standalone mode as opposed to managed mode (default)
+	deployStandalone bool
```

Comment:
Small naming suggestion to clarify that we are referring to the APM component and not the entire deployment:
```suggestion
	APMStandalone bool
```

See also the comment on the negation, maybe this should be `APMManaged` instead.